aclogview/Form1.cs

```diff
@@ -782,5 +813,51 @@ IEnumerable<TreeNode> GetTreeNodes(TreeNodeCollection nodes)
             }
         }
 
-    }
-}
+        private void CmdLock_Click(object sender, EventArgs e)
+        {
+            if (CmdLock.Text == "Lock")
+            {
+                CmdLock.Text = "UnLock";
+                listView_Packets.Enabled = false;
+            }
+            else
+            {
+                CmdLock.Text = "Lock";
+                listView_Packets.Enabled = true;
+            }
+
+        }
+
+        private void cmdforward_Click(object sender, EventArgs e)
+        {
+            if (listView_Packets.SelectedIndices.Count > 0)
+            {
+                int nextRow = listView_Packets.Items[listView_Packets.SelectedIndices[0]].Index + 1;

```

Comment:
You already check SelectedIndices.Count > 0 above this, so [0] should be valid here.